### PR TITLE
Add test for culturally sensitive restricted object

### PR DESCRIPTION
**Pull request by VivianChu**
#### Local Checklist
- [x] Tests written and passing locally?
- [x] Code style checked?
- [x] QA-ed locally?
- [x] Rebased with `master` branch?
- [ ] Configuration updated (if needed)?
- [ ] Documentation updated (if needed)?

#### What does this PR do?

Add test for culturally sensitive restricted object

@ucsdlib/developers - please review

### CODE REVIEW

**Comment by VivianChu** [comment]
@mcritchlow - We have a similar test for embargo object - https://github.com/ucsdlib/damspas/blob/feature/cultural_restricted_obj/spec/features/dams_object_spec.rb#L746
I added this because it was related to the cultural sensitive object issue for staging environment that Gabriela reported last week. bb3789396p, bb0000967b, bb9250195j  On staging, the display behavior was wrong for these objects.  I thought it was related to the solr index because it didn't match the rdf file update.  I added the tests to confirm it.  Today after Longshou updated the solr index for those objects, they looked fine.  There's no code need to be updated to support it.

File: spec/features/dams_object_spec.rb [hunk 1]
```diff
@@ -1468,3 +1468,32 @@
...
+    solr_index (@sensitiveObj.pid)
+  end
+  after(:all) do
+    @sensitiveObj.delete
+    @damsUnit.delete
+  end
+
+  it "should see the view content button" do
+    sign_in_developer
+    visit dams_object_path(@sensitiveObj.pid)
```
[the remark this answers is not in the document]
Comment (VivianChu) [reply spec/features/dams_object_spec.rb]: @lsitu - You're right.  It's missing.  I thought I added that.  I'll add that line.  Thanks

**Comment by VivianChu** [comment]
@lsitu - I don't add tests for public & curator user because for the existing embargo object, they don't have those tests for public & curator user.  I can add more tests to make it more complete.

**Comment by lsitu** [comment]
👍

**Pull request merged**